specs/mgt-file-picker.md

```diff
@@ -1,57 +1,75 @@
 # mgt-file-picker
 
-The file picker component provides a button, a drop down list of files, and as well as button to launch a full file browsing experience to enable a user to select a file from OneDrive or SharePoint for further action. This component uses the [mgt-file-list](./mgt-file-list.md) component to display files in the dropdown. The selected file object(s) is returned to the developer.
+The file picker component provides a button, a drop down list of files, and as well as button to launch a full file browsing experience to enable a user to select a file from OneDrive or SharePoint for further action. This component uses the [mgt-file](./mgt-file.md) components to display files in the dropdown. The selected file object(s) is returned to the developer.
 
 <img src="./images/mgt-file-picker.png"/>
 
 ## Supported functionality
 
 | Feature | Priority | Notes |
 | ------- | -------- | ----- |
+| **v1** | | |
 | Provide a button that renders a drop down when clicked | P0 | |	
-| Retrieve a list of files based on information provided | P0 | |
-| Use the `mgt-file-list` component to render the list inside of the drop down | P0 | |
-| On selection or deselection of a file, an event is fired | P0 | |	
+| Retrieve files based on specified insight type | P0 | |
+| Use `mgt-file` components to render each file inside of the drop down | P0 | |
+| On selection of a file, a `selectionChanged` event is fired | P0 | |	
 | The selected file object(s) is accessible by the developer for further action |P0 | |	
-| Ability for developer to toggle between single-select and multiselect | P1 |  | 
-| Visual indication of selected files | P2 | Will not be included in v1. |
-| Render the selected files | P2 | Will not be included in v1. We will provide a sample for developers to render selected files using `mgt-file` components themselves. |
-| Extended file picker is launched when “See all” is clicked | P2 | Not included in v1. |
+| Extended file picker is launched when “See all” is clicked | P0 | |
+| **v2** | | |
+| Retrieve files from a specified insight type or from a specific drive location based on identifiers provided by developer | P1 | |
+| Ability for developer to toggle between single-select and multiselect | P1 | Will not be included in v1. | 
```

Comment:
"Will not be included in v1" seems implied by being in the "v2" section. Could probably remove it.